src/components/Contentful/DynamicHero.vue

```diff
@@ -32,7 +32,7 @@
 								heroMedia[0].description,
 							]"
 						>
-							<kv-contentful-img
+							<kv-contentful-img-new
```

Comment:
Should this be `kv-contentful-img` ?

Reply:
(throughout)

Reply:
Well the `-new` represents sections that are using kv-components instead of our existing Ui based version.

I think I can go ahead with a full migration to use the kv-components version. It worked fine for the single image case which I've tested and I'll test the carousel too and get that using it.